structuralcodes/codes/mc2010/_concrete_creep_and_shrinkage.py

```diff
@@ -0,0 +1,818 @@
+    return abs(sigma / fcm)
+
+
+def calc_creep_coefficient(
+    time: np.ndarray,
+    t0: float,
+    T_cur: float,
+    fcm: float,
+    rh: float,
+    notional_size: float,
+    sigma: float,
+    agg_type: str,
+    cem_class: str,
+) -> np.ndarray:
+    """Calculate the creep coefficient.
+
+    Defined in fib Model Code 2010, Eqs. 5.1-63 and 5.1-64 - 5.1-71
+
+    args:
+        time (numpy.ndarray): The different times in days at which the
+            drying creep coefficient is determined.
+        t0 (float): The age of the concrete in days at which the
+            loading is applied.
+        T_cur (float): The temperature of the environment during curing
+            in degrees Celcius.
+        fcm (float): The mean compressive strength of the concrete in
+            MPa.
+        rh (float): The relative humidity of the environment.
+        notional_size (float): The notional size of the considered
+            element in mm, defined as 2A/h.
+        sigma (float): The compressive stress applied to the concrete
+            at ts in MPa.
+        agg_type (str): The type of aggregate used in the concrete.
+            The following values can be chosen:
+            ['Basalt', 'Quartzite', 'Limestone', 'Sandstone']
+        cem_class (str): The cement type that is used.
+            The choices are:
+                strength classes 32.5R and 42.5N;
+                strength classes 42.5R, 52.5N and 52.5R;
+                strength class 32.5N.
+
+    returns:
+        numpy.ndarray: The creep coefficient.
+    """
+    _check_age_at_loading(t0)
+    _check_fcm(fcm)
+    _check_RH(rh)
+    _check_initial_stress(sigma, fcm)
+    _check_agg_types(agg_type)
+    _check_cem_strength_class(cem_class)
+    TABULAR_VALUES = _get_creep_shrinkage_coeffs(cem_class)
+    t0_adj = _calc_modified_t0(t0, T_cur, TABULAR_VALUES)
+    # Calculate the basic creep coefficient (phi_bc)
+    # (see Eqs. 5.1-64 - 5.1-66 of [1]):
+    beta_bc_fcm = _calc_beta_bc_fcm(fcm)
+    beta_bc_t = _calc_beta_bc_t(time, t0, t0_adj)
+    phi_bc = calc_basic_creep_coefficient(beta_bc_fcm, beta_bc_t)
+    # Calculate the drying creep coefficient (phi_dc)
+    # (see Eqs. 5.1-67 - 5.1-71 of [1]):
+    beta_dc_fcm = _calc_beta_dc_fcm(fcm)
+    beta_dc_RH = _calc_beta_dc_RH(rh, notional_size)
+    beta_dc_t0 = _calc_beta_dc_t0(t0_adj)
+    alpha_fcm = _calc_alpha_fcm(fcm)
+    beta_h = _calc_beta_h(notional_size, alpha_fcm)
+    gamma_t0 = _calc_gamma_t0(t0_adj)
+    beta_dc_t = _calc_beta_dc_t(time, t0, beta_h, gamma_t0)
+    phi_dc = calc_drying_creep_coefficient(
+        beta_dc_fcm, beta_dc_RH, beta_dc_t0, beta_dc_t
+    )
+    # Calculate the creep coefficient (phi) (see Eq. 5.1-63)
+    phi = phi_bc + phi_dc
+    # Include the effect of high stress if needed (see Eq. 5.1-74 of [1]):
+    k_sigma = _calc_k_sigma(sigma, fcm)
+    if 0.4 <= k_sigma <= 0.6:
+        return np.exp(1.5 * (k_sigma - 0.4)) * phi
+    return phi
+
+
+def calc_creep_compliance(
```

Comment:
The signature should have only Eci(t0), Eci and phi(t,t0). We have the functions to compute them.
As before the logic of this function can be used for the concreteMC2010 class? Or we could think about a further function here that uses this logic That is additional to the small equation that computes the second term of 5.1-61